mp/src/public/tier1/convar.h

```diff
@@ -355,6 +355,18 @@ friend class ConVarRef;
     FORCEINLINE_CVAR char const *GetString(void) const;
     FORCEINLINE_CVAR Color GetColor(void) const;
 
+    FORCEINLINE_CVAR void Nuke()
```

Comment:
Can we call this `Unload()` instead? This doesn't really nuke anything, it just shuts it down.